### PR TITLE
Add unread and date separators

### PR DESCRIPTION
**Pull request by diegolmello**
screen shot 2019-01-21 at 16 40 50
screen shot 2019-01-21 at 17 59 17

### CODE REVIEW

**Comment by renatobecker** [comment]
@diegolmello,

When there is an unread messages alert and the Livechat widget is minimized and then maximized, the position of the alert component is changing:

screen shot 2019-01-22 at 01 23 17

File: src/components/App/index.js [hunk 1]
```diff
@@ -17,6 +17,18 @@ import GDPRAgreement from '../../routes/GDPRAgreement';
 import Register from '../../routes/Register';
 import { Provider as StoreProvider, Consumer as StoreConsumer } from '../../store';
 
+let hidden = null;
```
Comment (renatobecker) [inline src/components/App/index.js]: It would be better moving this code/logic to `helper` library.

**Comment by renatobecker** [comment]
@tassoevan, can you please help us with the review process in this PR? Your opinion / review would be very important.
Thanks.

File: src/routes/Chat/container.js [hunk 2]
```diff
@@ -212,7 +221,36 @@ export class ChatContainer extends Component {
...
+		const { messages, visible } = this.props;
+
+		// show unread messages alert, e.g. "1 new message since..."
+		if (nextProps.visible && visible !== nextProps.visible && nextProps.messages.length > 0 && lastReadMessageId) {
+			const { alerts, dispatch } = this.props;
+			const lastReadMessageIndex = nextProps.messages.findIndex((item) => item._id === lastReadMessageId);
+			const unreadMessages = nextProps.messages.slice(lastReadMessageIndex + 1);
+			if (unreadMessages.length > 0) {
+				const lastReadMessage = nextProps.messages[lastReadMessageIndex];
+				const message = `${ unreadMessages.length } new ${ unreadMessages.length === 1 ? 'message' : 'messages' } since ${ format(lastReadMessage.ts, 'HH:mm [on] MMMM Do') }`;
```
Comment (ggazzo) [inline src/routes/Chat/container.js]: i18n

File: src/components/Separator/styles.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    &__line {
+        display: flex;
+        flex: 1;
+        border: 0;
+        border-top-width: 2px;
+        border-style: solid;
+    }
+
+    &--date {
```
Comment (ggazzo) [inline src/components/Separator/styles.scss]: mixin

File: src/components/Separator/styles.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+@import '~styles/variables';
+
+.separator {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: $default-gap 0;
+
+    &__text {
+        height: 16px;
```
Comment (ggazzo) [inline src/components/Separator/styles.scss]: use variables

**Comment by ggazzo** [comment]
@diegolmello @tassoevan I think tasso will finish this pr, its almost perfect just few changes (:

**Comment by tassoevan** [comment]
Closes #130.

**Pull request merged**